ui/packages/catalog-ui-search/src/main/webapp/js/model/Query.js

```diff
@@ -248,7 +248,7 @@ Query.Model = Backbone.AssociatedModel.extend({
     }
     var query = this
     this.currentSearches = sources.map(function(src) {
-      data.src = src
+      data.srcs = [src]
```

Comment:
Check for other places that may need to be updated to send an array of strings instead of a single string